### PR TITLE
hw-mgmt: patches: 5.10: Update LED and fan patches

### PR DESCRIPTION
**Pull request by felixradensky**
Reviewed-by: Vadim Pasternak <vadimp@nvidia.com>

### CODE REVIEW

**Comment by felixradensky** [comment]
RERUN

File: recipes-kernel/linux/linux-5.10/0309-hwmon-mlxreg-fan-Add-support-for-new-flavour-of-capa.patch [hunk 1]
```diff
@@ -27,14 +28,41 @@ bitmask or counter.
...
+ 	u32 prsnt;
++	u32 shift;
+ };
+ 
+ /*
+@@ -146,8 +148,10 @@ mlxreg_fan_read(struct device *dev, enum hwmon_sensor_types type, u32 attr,
+ 				/*
+ 				 * Map channel to presence bit - drawer can be equipped with
+ 				 * one or few FANs, while presence is indicated per drawer.
++				 * Shift channle value if necessary to align with register value.
```
Comment (sholeksandr) [inline recipes-kernel/linux/linux-5.10/0309-hwmon-mlxreg-fan-Add-support-for-new-flavour-of-capa.patch:57]: channle -> channel

**Review by vadimp-nvidia — APPROVED** [review]

File: recipes-kernel/linux/linux-5.10/0309-hwmon-mlxreg-fan-Add-support-for-new-flavour-of-capa.patch (hunk 1 above)
Comment (felixradensky) [reply recipes-kernel/linux/linux-5.10/0309-hwmon-mlxreg-fan-Add-support-for-new-flavour-of-capa.patch]: Thanks, fixed

**Comment by felixradensky** [comment]
RERUN

**Comment by felixradensky** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

**Comment by felixradensky** [comment]
RERUN

**Pull request merged**